Week-3/assignment3/jsArray.js

```diff
@@ -0,0 +1,25 @@
+const divideAr = (ar) => {
+  let even = ar.filter((item) => item % 2 === 0);
+  let odd = ar.filter((item) => item % 2 !== 0);
+  even.sort(function compareNumbers(a, b){return a - b;});
+  odd.sort(function compareNumbers(a, b){return a - b;});
```

Comment:
Here
`function compareNumbers(a, b){return a - b;}` this can be replaced with `(a, b) => a-b`.

Thanks